src/status_im/chat/events/input.cljs

```diff
@@ -283,6 +282,8 @@
                          :chat-id      chat-id
                          :jail-id      (or owner-id jail-id)
                          :content      {:command (:name command)
+                                        :scope   (when-not (:to-message-id metadata)
```

Comment:
What's the purpose of this expression, why is :scope nil when `:to-message-id` is present in metadata ?

Reply:
Because we don't use scopes for requests and `:to-message-id` is the indication of request

Reply:
So we have no better way of distinguishing command-request message from command message other then inspecting `:to-message-id` key in the metadata ?